bingo/bingo-elastic/python/bingo_elastic/model/record.py

```diff
@@ -120,8 +135,8 @@ def as_dict(self) -> Dict:
         }
 
     def as_indigo_object(self, session: Indigo):
-        assert self.cmf
-        return session.deserialize(list(map(int, self.cmf.split(" "))))
+        assert self.cmf != ""
```

Comment:
@AATDev21 @mkviatkovskii @khyurri  I could think of this assert only if we aren’t sure that cml variable could be a string?

Reply:
And in cases, when we expect it to be an actual empty string, catch deserialization error somehow else? I think the purpose of this assert originally was close to it.

Reply:
`assert self.cmf` and `assert self.cmf != ""` looks messy

I'd prefer something like that
```
def as_indigo_object(self, session: Indigo):
    if self.cmf:
        return session.deserialize(list(map(int, self.cmf.split(" "))))
    raise ValueError("Unexpected cmf value")
```

Reply:
Fixed, thank you, Ruslan!